src/api/.rubocop.yml

```diff
@@ -81,3 +81,311 @@ Rails/BulkChangeTable:
     - 'db/migrate/20170912140257_change_kiwi_packages_columns_from_big_int_to_int.rb'
     - 'db/migrate/20171107125828_rename_kiwi_preference_types_to_kiwi_preferences.rb'
     - 'db/migrate/20180516074538_explicitly_set_charset.rb'
+
+Style/MethodCallWithArgsParentheses:
+  Enabled: true
+  Exclude:
+    - 'app/views/*'
+  IgnoredMethods:
```

Comment:
How was this list of ignored methods defined? I would add a comment to describe this.

Reply:
done

Reply:
I would prefer to have actually two cops. One for ```spec``` and ```test``` directories and other for the rest, with that approach we can reduce the number of methods in the whitelist and avoid missing cases like:

```
mail = Mail.new
...
mail.to 'foo@example.org' 
```
in a controller shouldn't be allowed, however a 

```
it { expects(response).to be_success
```

should be ok.

what do you think?

Reply:
@coolo: Where is the comment?

Reply:
in .rubocop.yml?

Reply:
```
class MyClass 
  def get(foo)
   ...
  end
```

```MyClass.new.get 'foo' ``` would bypass the Cop, having a huge list of cops in one place, but not if we have two different rubocop configs (one for ```app``` dir and other just for ```spec``` and ```test``` dirs)